app/Models/User.php

```diff
@@ -1104,6 +1111,13 @@ public function setPlaymodeAttribute($value)
         $this->osu_playmode = Beatmap::modeInt($value);
     }
 
+    public function hasBlocked(self $user)
```

Comment:
`static`? :zap:

Reply:
can't use `static` as a param type in php, go figure